src/documenteer/ext/bibtex.py

```diff
@@ -19,14 +26,84 @@
 
 from ..version import __version__
 
-__all__ = ["setup"]
+
+@node
+def truncated_names(
+    children: list[Node], context: typing.Any, role: str, **kwargs: typing.Any
+) -> Node:
+    """Return formatted names, truncating long author lists."""
+    assert not children  # noqa: S101 (matches pybtex original)
+
+    try:
+        persons = context["entry"].persons[role]
+    except KeyError:
+        raise FieldIsMissing(role, context["entry"]) from None
+
+    # If there are more than 7 authors we return the first 3.
+    if len(persons) > 7:
+        persons = persons[:3] + [Person("others")]
+    style = context["style"]
+    formatted_names = [
+        style.format_name(person, style.abbreviate_names) for person in persons
+    ]
+    return join(**kwargs)[formatted_names].format_data(context)
 
 
 class LsstBibtexStyle(pybtex.style.formatting.plain.Style):
     """Bibtex style that understands ``docushare`` fields in LSST
     bibliographies.
     """
 
+    # Allows a mapping of journal or series name found in the bibtex file
+    # to a more usable name for the Sphinx output. In particular ADS returns
+    # bib entries that assume latex commands exist to convert from \apj
+    # to ApJ or Astrophys. J as needed (allowing each style to decide how
+    # to report journal names).
+    bib_name_to_usable_name: typing.ClassVar = {
```

Comment:
A more descriptive variable name?

```suggestion
    bib_to_display_name: typing.ClassVar = {
```